app/views/assignments/_assignment.html.slim

```diff
@@ -5,5 +5,5 @@ tr
   td = t("state.#{assignment.state}", scope: [:simple_form, :options, :assignment]) if assignment.state.present?
   td = assignment.created_at.strftime("%d-%m-%Y")
   td = link_to assignment.creator, assignment.creator if assignment.creator.present?
-  = render 'index_actions', subject: assignment
+  = render 'index_actions', subject: assignment, controller_name: 'assignments'
```

Comment:
This is used only to overwrite the controller_name at the  app/views/application/_index_actions.html.slim template. I need that because otherwise there is the same problem described above.